index.html

```diff
@@ -352,5 +330,83 @@ <h2 class="text-center">Skills</h2>
         </div>
       </div>
     </section>
+
+    <section class="contact-section relative">
+      <div class="contact-header">
+        <h2 class="text-center">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+      <div class="contact-form">
+        <form action="#" class="flex flex-col">
```

Comment:
kindly use an action to collect the data
`<form action="https://formspree.io/f/{form_id}" method="post">`